.gitignore

```diff
@@ -1,11 +1,3 @@
 .vscode
 *.o
-all
-binhex/binhex
-comm/frommac
-comm/tomac
-frommac/frommac
-hexbin/hexbin
-macunpack/macunpack
-mixed/macsave
-mixed/macstream
\ No newline at end of file
+/builddir
```

Comment:
FWIW, meson doesn't need this at all, at least for modern versions of Meson: https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

Reply:
Good to know!